pages/linux/smbpasswd.md

```diff
@@ -0,0 +1,19 @@
+# smbpasswd
+
```

Comment:
Missing description of the command. 
Also a link if we can find one.

Reply:
Isn't the description on line 3 ? Did you mean something else ?

Reply:
Oh yes. My mistake.